app/views/documents/show/_requirements.html.erb

```diff
@@ -1,13 +1,13 @@
 <% issue_params = {
   style: "summary",
   link_options: {
-    title: { href: "#content" },
-    summary: { href: "#content" },
-    body: { href: "#content" },
-    change_note: { href: "#content" },
-    alt_text: { href: "#lead-image" },
-    caption: { href: "#lead-image" },
-    topics: { href: "#topics" }
+    title: { href: edit_document_path(@edition.document, anchor: "edit-document-title") },
+    summary: { href: edit_document_path(@edition.document, anchor: "edit-document-summary") },
+    body: { href: edit_document_path(@edition.document, anchor: "edit-document-body") },
+    change_note: { href: edit_document_path(@edition.document, anchor: "edit-document-change-note") },
+    alt_text: @edition.lead_image_revision == nil ? nil : { href: update_image_path(@edition.document, @edition.lead_image_revision.image_id, wizard: params[:wizard], anchor: "edit-image-alt-text") },
+    caption: @edition.lead_image_revision == nil ? nil : { href: update_image_path(@edition.document, @edition.lead_image_revision.image_id, wizard: params[:wizard], anchor: "edit-image-caption") },
```

Comment:
Ah nicely done!

You can also do `@edition.lead_image_revision.nil?` rather than needing an equality check. To be honest though, since https://github.com/alphagov/content-publisher/pull/1086 will resolve these issues I'd cut them out and have this PR not handle any images.